tensorboard/webapp/plugins/npmi/store/npmi_selectors.ts

```diff
@@ -52,3 +51,17 @@ export const getRunToMetrics = createSelector(
     return state.runToMetrics;
   }
 );
+
+export const getMetricFilterKeys = createSelector(
+  selectNpmiState,
+  (state: NpmiState): string[] => {
+    return Object.keys(state.metricFilters);
+  }
+);
+
+export const getMetricsRegex = createSelector(
```

Comment:
I'm curious why the `metricsRegex` needs to be in the ngrx store, rather than just local state in the component.  One reason I could imagine is to ensure the user-entered text stays in the input, even when they move to another dashboard, then return to npmi later.  For my own understanding, are there other reasons?

Reply:
No, no other reasons. Still thought this would make sense, should I change it?